### DISCUSSION TITLE
TokenJWT

### DISCUSSION THREAD

**Discussion by JediThumper** [SchneiderInfosystems/FB4D#151] [Q&A] [answered]
Hi
I'm starting out with FB4D.

I've gone through the demo app and can get everything working fine. 

No I am trying on my own small test project but everything inside the {$IFDEF TOKENJWT} is grayed out. In the Demo its fine.

All the paths are there, I can see the  JOSE.Core.JWT and no errors are shown. What am I missing?

Many thanks

Alan

**Comment by JediThumper** [comment]
I added a Conditional Define in the project options
but Now in unit FB4D.OAuth
I get an error: [dcc32 Error] FB4D.OAuth.pas(35): E2003 Undeclared identifier: 'ITokenJWT'

**Comment by JediThumper** [comment]
OK so I had to manually add all the FB4D pas files to the project.
Now compiles

**Comment by SchneiderInfosystems** [comment, marked as answer]
I highly recommend starting with this wiki page:
https://github.com/SchneiderInfosystems/FB4D/wiki/Getting-Started-with-FB4D
There exactly such questions should be clarified step by step from the beginning. 
I wish you much success with this library. 
Remember that this project cannot be sustained without sponsors. So if you build a commercial product, I recommend you to donate for further maintenance.
